cmd/config/constants.go

```diff
@@ -33,6 +33,10 @@ const (
 	EnvPublicIPs    = "MINIO_PUBLIC_IPS"
 	EnvEndpoints    = "MINIO_ENDPOINTS"
 
+	// API sub-system
+	EnvAPIConnMax      = "MINIO_API_CONN_MAX"
+	EnvAPIConnDeadline = "MINIO_API_CONN_DEADLINE"
```

Comment:
Since you are already discussing, IMO it should be 'requests' and not 'connections'. People may think this is somehow related to the number of open TCP connections, which it is not.

```suggestion
	EnvAPIConnMax      = "MINIO_API_REQUESTS_MAX"
	EnvAPIConnDeadline = "MINIO_API_REQUESTS_DEADLINE"
```

Reply:
This is already decided @klauspost keeping it as _CONN for now because it says its API connections, requests is also fine will check with @abperiasamy